perf/impl/go-libp2p/v0.29/main.go

```diff
@@ -99,7 +102,10 @@ func main() {
 }
 
 type Result struct {
-	Latency float64 `json:"latency"`
+	Type          string  `json:"type"`
+	TimeSeconds   float64 `json:"timeSeconds"`
+	UploadBytes   uint    `json:"uploadBytes"`
```

Comment:
This would better be a uint64, otherwise you'll quickly run into limits on 32 bit machines.